docs/arch/adr-27.md

```diff
@@ -0,0 +1,53 @@
+# ADR 27: Subject Viewer Config
+
+Created: June 26, 2020
+
+## Context
+
+A workflow determines which subject viewer to use with the `workflow.configuration.subject_viewer` property. Subject viewers include `singleImage`, `lightCurve`, `multiFrame`, and `subjectGroup` at creation of this ADR. Some subject viewers can utilize, or require, additional configuration information.
+
+### [multiFrame](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer)
+
+A workflow using the multi-frame subject viewer might have a preference regarding:
+marks per frame: some might prefer marks filtered per frame, like a transcription workflow where each frame represents a unique page to transcribe with marks only relevant to each page, while other workflows might prefer marks persist between frames [, like ?] (`multi_image_clone_markers` in PFE)
+pan and zoom: some might prefer pan and zoom reset per frame, like a transcription workflow where each frame represents a unique page to transcribe, while other workflows might prefer pan and zoom maintained between frames, like [Wildcam Gorongosa](https://www.zooniverse.org/projects/zooniverse/wildcam-gorongosa/classify) or [Backyard Worlds](https://www.zooniverse.org/projects/marckuchner/backyard-worlds-planet-9/classify) (in flipbook mode, not separate frames)
```

Comment:
Maybe bullet point here? Is there a current setting for this on PFE? We should note that here or is a certain function assumed when using flipbook vs display all frames at once?